library/common/strings/string_conversions.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <string>
+
+std::string UTF16ToUTF8(const char16_t* utf16, size_t len);
```

Comment:
nit: Comments for these methods, pelase.

Reply:
Done (sorry about that).